docs/install_Device_Northbound.md

```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
 
-2. Add Topic and Permission
+ ![DataXess1](graphics/data1.png)
 
-- Topic: `ie/#`
-- Permission: `Publish and Subscribe`
+3. Start the configuration by creating a new group
+   
+  ![DataXess2](graphics/data2.png)
 
-![ie_mqtt_Connector](graphics/IE_MQTT_Connector_Topic.png)
-    
-![ie_mqtt_Connector](graphics/MQTT_Connector.png)
+4. Add a group name and description and add all 3 devices to it. Make sure to select the central device as the "Aggregator Device" and click "Save".
+   
+  ![DataXess3](graphics/data3.png)
 
-3. Set "Unsecure" in the "Certificate" settings
+5. Click on one of the setting symbols of your new created group to configure the data that get's exchanged. Select the topics `ie/d/j/simatic/v1/iefc/dp/r/#` and `ie/m/j/simatic/v1/iefc/dp` for both acquistion devices.
+   
+  ![DataXess4](graphics/data4.png)
+  ![DataXess5](graphics/data5.png)
 
-![IE_MQTT_Connector](graphics/IE_MQTT_Connector_Certificate.png)
+6. DataXess will automatically create new topics on the aggregator device were the data will be published.
+   
+  ![DataXess6](graphics/data9.png)
 
-4. Bridge Configure
+7. Click "Deploy".
+  
+ ![DataXess6](graphics/data6.png)
+   
+8. Configure a new user that will be used to subscribe and publish to the configured topics and click "Validate",
+   
+  ![DataXess7](graphics/data7.png)
 
-   - Insert User: `edge`
-   - Insert password: `edge`
-   - Select Topic:  `ie/#`
-   - Direction: `IE MQTT Connector` :arrow_right: `IE Databus`
+If everything was configured correctly, all devices should have a green status. Also, the general connection status msut be in green as well, as shown:
 
-![IE_MQTT_Connector](graphics/IE_MQTT_Connector_Bridge.png)
+  ![DataXess8](graphics/data8.png)
 
-##  IIH Insights Hub Sync
+##  Insights Hub Sync
 
-All the following steps for this option will be performed in the "IIH Configurator" app on the Central Device.
+In this section, the necessary configurations will be made to establish communication with the cloud (Insights Hub).
 
-### Integrate Data Service
+All the following steps for this option will be performed in the "Common Configurator" app on the Aggregation Device (Shopfloor-to-cloud).
 
-To store data in the Industrial Information Hub (IIH), it is required to integrate the Data Service Application. 
+### Integrate IIH Essentials into Common Configurator
 
-1. Go to the "Store Data" tab
+1. Go to *Edge Device UI > Apps*, open *Common Configurator*. Then, go to *Store Data*.
 
 2. Click on "Integrate"
    
-  ![IIH_IntegrateDataService](graphics/IIH_IntegrateDataService.png)
+  ![IIH_Integrate](graphics/IIH1.png)
 
 3. Confirm the integration in the popup window
 
-  ![IIH_IntegrateDataServiceConfirmation](graphics/IIH_IntegrateDataServiceConfirmation.png)
+  ![IIH_IntegrateConfirmation](graphics/IIH2.png)
+
+### Configure Connection to Insights Hub.
 
-### Configure Connections
+Common Configurator needs a connection to the Databus and to Insights Hub. 
```

Comment:
Especially IIH Essentials needs a connection to Databus for colecting data

Reply:
The point of that statement is to say that Common Configurator (with IIH Essentials integrated as shown in the previos part) need Databus and Insights Hub.